packages/block-library/src/legacy-widget/edit/index.js

```diff
@@ -55,7 +62,7 @@ class LegacyWidgetEdit extends Component {
 
 			if ( ! hasPermissionsToManageWidgets ) {
 				placeholderContent = __( 'You don\'t have permissions to use widgets on this site.' );
-			} else if ( availableLegacyWidgets.length === 0 ) {
+			} else if ( visibleLegacyWidgets.length === 0 ) {
```

Comment:
This will always be false as `visibleLegacyWidgets` is an object meaning `visibleLegacyWidgets.length` is always `undefined`.

Reply:
Corrected 👍